sources/engine/Stride.Input/SDL/PointerSDL.cs

```diff
@@ -62,25 +62,24 @@ private void OnSizeChanged(WindowEvent eventArgs)
             SetSurfaceSize(new Vector2(uiControl.ClientSize.Width, uiControl.ClientSize.Height));
         }
 
-        // TODO Code from PointeriOS with slight modifications, consider creating and referencing a utility class
```

Comment:
Did you mean to remove this TODO comment?

Reply:
Yes, because the code is different now. I imagine this issue doesn't exist at all on iOS.